packages/depot_tools.rb

```diff
@@ -23,26 +23,24 @@ def self.install
       FileUtils.rm_rf 'man/src'
       FileUtils.rm_rf Dir.glob('.git*')
       system 'find -name \'*.bat\' -exec rm {} +'
-      FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/depot_tools"
-      FileUtils.mv 'man/', "#{CREW_DEST_PREFIX}/share"
-      FileUtils.cp_r '.', "#{CREW_DEST_PREFIX}/share/depot_tools"
+      FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/libexec/depot_tools"
+      FileUtils.mv 'man/', "#{CREW_DEST_PREFIX}/libexec"
+      FileUtils.cp_r '.', "#{CREW_DEST_PREFIX}/libexec/depot_tools"
       FileUtils.mkdir_p "#{CREW_DEST_HOME}/.config/.vpython-root"
       FileUtils.mkdir_p "#{CREW_DEST_HOME}/.config/.vpython_cipd_cache"
       FileUtils.ln_s "#{HOME}/.config/.vpython-root/", "#{CREW_DEST_HOME}/.vpython-root"
       FileUtils.ln_s "#{HOME}/.config/.vpython_cipd_cache/", "#{CREW_DEST_HOME}/.vpython_cipd_cache"
     end
+
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/env.d/"
+    @env = <<~EOF
+      # Add depot-tools to path
+      PATH="$PATH:#{CREW_PREFIX}/libexec/depot_tools"
```

Comment:
Change from libexec back to share?

Reply:
done. it would make more sense, according to the FHS to be in libexec since they are executables.